ui/app/common/domain/services/locationService.js

```diff
@@ -28,10 +28,19 @@ angular.module('bahmni.common.domain')
             });
         };
 
+        var getFacilityVisitLocation = function (locationUuid) {
```

Comment:
@kavitha-sundararajan can you please add tests for the new function

Reply:
added tests